apps/webview/src/App.tsx

```diff
@@ -1,15 +1,35 @@
+import { BrowserRouter as Router, Routes, Route} from 'react-router-dom';
 import React from "react";
-import './assets/css/global/page.css'
-import SwiperOnboard from "./assets/pages/onboarding/onboard";
+import './page.css'
+import SwiperOnboard from "./pages/onboarding/onboard";
+import Dashboard from "./pages/dashboard/dashboard";
+import OnboardingLast from './pages/onboarding/onboard_last';
+import LoginApp from './pages/auth/login';
+import RegisterApp from './pages/auth/register';
+import { ProtectedRoute }  from './components/protectedRoute';
+import { AuthProvider } from "./hooks/useAuth";
+
 const App: React.FC = () => {
   return (
-    <>
-    <div className="canvas_mobile">
-      <div className="mobile_view">
-        <SwiperOnboard/>
+      <div className="canvas_mobile">
+        <div className="mobile_view">
+          <Router>
+            <AuthProvider>
+              <Routes>
+                <Route path="/" element={<SwiperOnboard />} />
+                <Route path="/onboarding-last" element={<OnboardingLast />} />
+                <Route
+                  path="/dashboard"
+                  element={
+                  <ProtectedRoute element={<Dashboard />} />
```

Comment:
Can you explain your implementation of protected routes here?

Reply:
I used custom created hooks useAuth and useLocalStorage to verify the token stored in local storage, to keep a user logged in. I used [this blog](https://blog.logrocket.com/authentication-react-router-v6/) as a reference.

Reply:
This implementation works too. 👍